pkg/cli/common.go

```diff
@@ -119,9 +119,13 @@ var (
 			Name:  "label",
 			Usage: "Set metadata for an image (default [])",
 		},
+		cli.BoolFlag{
+			Name:  "layers",
+			Usage: "cache intermediate layers during build.",
```

Comment:
Could you add an environment variable BUILDAH_LAYERS and then have bool function useLayers() that checks if this is set to true/false (Default to false if not set)  Then set the default value here
Value: useLayers()

Reply:
Added the env variable.

Reply:
You need to add
```
Value: useLayers()
```
Here.

Then if I do buildah bud --help
It will show me the default as true or false, depending on if the environment is set.